src/main/steps/c100-rebuild/child-details/live-with/living-arrangements/postController.ts

```diff
@@ -37,6 +38,12 @@ export default class ChildLivingArrangementsPostController extends PostControlle
       req.session.userCase.cd_children
     ) as ChildrenDetails[];
 
+    req.session.userCase.oprs_otherPersons?.forEach(person => {
```

Comment:
req.session.userCase?.oprs_otherPersons

Reply:
This optional check can't be moved because userCase can't be undefined, but oprs_otherPersons is optional.